.template/addons/github/.github/workflows/publish_wiki.yml.tt

```diff
@@ -2,17 +2,35 @@ name: Publish Wiki
 
 on:
   push:
-    paths:
-      - .github/wiki/**
     branches:
       - develop
+    paths:
+      - .github/wiki/**
+
+env:
+  USER_NAME: github-wiki-workflow
+  USER_EMAIL: bot@nimblehq.co
 
 jobs:
   publish:
-    name: Publish Wiki
-    uses: nimblehq/github-actions-workflows/.github/workflows/publish_wiki.yml@0.1.0
-    with:
-      USER_NAME: github-wiki-workflow
-      USER_EMAIL: ${{ secrets.GH_EMAIL }}
-    secrets:
-      USER_TOKEN: ${{ secrets.GH_TOKEN }}
+    name: Publish Github Wiki
+    runs-on: ubuntu-latest
+    timeout-minutes: 1
+
+    steps:
+      - name: Cancel previous runs
+        uses: styfle/cancel-workflow-action@0.8.0
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: Publish Github Wiki
+        uses: nimblehq/publish-github-wiki-action@v1.0
+        with:
+          user_name: ${{ env.USER_NAME }}
+          user_email: ${{ env.USER_EMAIL }}
+          user_access_token: ${{ secrets.USER_ACCESS_TOKEN }}
```

Comment:
According to the insight you provided:

> with section wasn't getting access to secrets context.

it seems that this line would not be able to access `secrets. USER_ACCESS_TOKEN`, is it? 👀 

If not, we might want to use the `env` technic for it as well. 💭 

```suggestion
          user_access_token: ${{ env.USER_ACCESS_TOKEN }}
```

Reply:
I think it's a different one. As previously, we delegate our `publish action` to another `workflow` -> so we cannot pass the value to the `with` section from the `secrets`. And, currently, we're moving on delegating the `publish action` to an `action` (not workflow anymore), so we can use it directly.

Long shared that interesting information with me last Friday, so if you want to dive deeper into that topic, feel free to ping him 😆😆😆

Reply:
Hi @malparty, 

I ran [this](https://github.com/mosharaf13/rails-template-wiki/actions/runs/5485979793) action using [this](https://github.com/mosharaf13/rails-template-wiki/actions/runs/5485979793/workflow) workflow file. It completed successfully. I am updating the PR description for future reference.

Reply:
hehe, interesting then that means we do not need the `env` section at all, no? (or maybe I misunderstood something? 🙈 )

Reply:
Thanks 👍🏼 . Fixed in f68814abcb4b2b56cbc8a26fea64e0b865337766